File: vs/Listas/Lista 2/media.c
#include <stdio.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL, "Portuguese"); // Declaração de idioma usado, para utilização de acentos.
    float n1=0,n2=0,n3=0,med=0;
    
	printf("Insira as 3 notas n1, n2 e n3 para realizar o cálculo da média final:\n");
	scanf("%f %f %f",&n1,&n2,&n3);
	
	med=((n1*2)+(n2*3)+(n3*5))/10;
	
	if(n1>100||n2>100||n3>100||n1<0||n2<0||n3<0){
		printf("\nAlguma nota inserida não está entre 0 e 100. Insira-as novamente\n");
		main();
	}
	
	if (med>=90 && med<=100){ 
		printf("\nSua nota foi %.2f. Otimo resultado!\n",med);
	}
	
	else if(med>=70&&med<90){
		printf("\nSua nota foi %.2f. Bom resultado!\n",med);
	}
	
	else if(med>=50&&med<70){
		printf("\nSua nota foi %.2f. Resultado regular.\n",med);
	}
	
	else if(med>=0&&med<50){
		printf("\nSua nota foi %.2f. Resultado ruim.\n",med);
	}
	
    return 0;   
}

File: vs/Listas/Lista 2/media.py
def med():
    n1=float(input("\nInsira um valor para a nota 1, com peso 2:  "))
    n2=float(input("\nInsira um valor para a nota 2, com peso 3:  "))
    n3=float(input("\nInsira um valor para a nota 3, com peso 5:  "))

    if n1>100 or n2>100 or n3>100 or n1<0 or n2<0 or n3<0:
        sn=input("Alguma nota inserida nao esta entre 0 e 100. Deseja comecar de novo?[S/N]")
        if sn == 's' or sn == 'S':
            med()
        else:
            return
            
    media=((n1*2)+(n2*3)+(n3*5))/10

    if media>=90 and media<=100:
        print("\nOtimo resultado! Sua nota foi: ")
        print(media)
    elif media>=70 and media<90:
        print("\nBom resultado! Sua nota foi: ")
        print(media)
    elif media>=50 and media<70:
        print("\nResultado regular. Sua nota foi: ")
        print(media)
    elif media>=0 and media<50:
        print("\nResultado ruim. Sua nota foi: ")
        print(media)
        
med()

